tests/CppUTestExt/MockNamedValueTest.cpp

```diff
@@ -27,6 +27,7 @@
 
 #include "CppUTest/TestHarness.h"
 #include "CppUTestExt/MockNamedValue.h"
+#include <cfloat>
```

Comment:
Is it possible to get rid of the cfloat include here? We try to avoid direct Standard C includes to increase portability.

Reply:
Absolutely. It's not used, I just forgot to remove it. Will fix once I'm back from holiday.